packages/engine/src/experiment/controller/controller.rs

```diff
@@ -302,6 +302,11 @@ impl<P: OutputPersistenceCreatorRepr> ExperimentController<P> {
 impl<P: OutputPersistenceCreatorRepr> ExperimentController<P> {
     pub async fn run(mut self) -> Result<()> {
         let mut terminate_recv = self.terminate_recv.take_recv()?;
+
+        let mut waiting_for_completion = None;
+        let mut time_to_wait = 1;
```

Comment:
On reflection the initial `time_to_wait` should be more than 1 second (maybe 5 seconds?), because waiting for 1 second is typical and not worth warning about.

Reply:
It won't be logged on the first one, as the branch about `waiting_for_completion` will only happen after it's been set (and awaited) the first time.

Reply:
I see, we can leave it then